idea/gearexe/frontend/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "gearexe-idea-frontend",
+  "private": true,
+  "version": "0.0.1",
+  "type": "module",
+  "scripts": {
+    "start": "npx vite",
+    "build": "rm -rf dist && npx tsc -b && npx vite build",
```

```suggestion
    "build": "rm -rf dist && npx vite build",
```

With enabled `vite-plugin-checker` there's no need for `tsc`.